### ISSUE TITLE
[NIGHTWACH E2E]: Claims & Appeals - Progressbar must be updated for axe violation in Applications / Letters / Tests / 01 Authed E2e Spec

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21797]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe violations were discovered where ARIA progressbar attributes were not conforming to valid values. Preliminary research indicates this was for a custom version of the [Design System loading indicator](https://design.va.gov/storybook/?path=/story/components-loadingindicator--default).

## Related Issue:
* [[AUTOMATION]: Update Nightwatch axeCheckJS to capture failing tests ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/21738)

## Acceptance Criteria
- [ ] All instances of the error are fixed
- [ ] Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations
- [ ] Nightwatch e2e test `01 Authed E2e Spec` passes axeCheck validation OR
- [ ] Test is converted to Cypress and passes axeCheck validation

## Environment
* src/applications/letters/tests/01-authed.e2e.spec.js

**Comment by 1Copenut** [comment]
@vhenry07 Hello! When you have a few minutes, would you mind taking a look at this issue? It's a blocker for some work I'm doing to fix an issue in our Nightwatch accessibility test helper. Thank you.

**Comment by Mottie** [comment]
Sorry for the delay, I just merged in the change that cleaned up the rest of the un-used aria labels.

**Comment by 1Copenut** [comment]
NP! I'm bouncing around between a few things this past week.

**Comment by 1Copenut** [comment]
No issues remaining on this ticket. The Nightwatch tests are passing in my WIP branch with no regression. Moving to closed.
